pyproject.toml

```diff
@@ -36,16 +36,20 @@ skip_glob = '''
 
 [tool.pytest.ini_options]
 addopts = '''
-    -sv
-    --cov=. 
-    --cov-report=html:report/html
-    --cov-report=xml
-    --cov-report=term-missing 
-    --cov-fail-under=100 
     --last-failed 
     --last-failed-no-failures all
 '''
 
 [tool.coverage.run]
 relative_files = true
 omit = ["tests/**", "docs/**", "contrib/**", "snap/**", "setup.py"]
+
+[tool.coverage.report]
+fail_under = 100
+show_missing = true
+
+[tool.coverage.html]
+directory = "tests/unit/report/html"
+
+[tool.coverage.xml]
+output = "tests/unit/report/coverage.xml"
```

Comment:
sonar-project.properties:sonar.python.coverage.reportPaths should point here

Reply:
Path changed in `sonar-project.properties`